iaso/api/org_unit_types/serializers.py

```diff
@@ -9,6 +10,20 @@
 from ..projects.serializers import ProjectSerializer
 
 
+def get_parents(type_id):
+    parents_ids = []
+    if type_id is not None:
+        queryset = OrgUnitType.objects.filter(sub_unit_types__id=type_id)
+        for parent in queryset.all():
+            if parent.id not in parents_ids:
+                parents_ids.append(parent.id)
+            great_parents = get_parents(parent.id)
+            for great_parent_id in great_parents:
+                if great_parent_id not in parents_ids:
+                    parents_ids.append(great_parent_id)
+    return parents_ids
```

Comment:
@olethanh this can probably be improved, also we should write test for this .. it could be a good pair programing topic ?